octodns_edgecenter/__init__.py

```diff
@@ -219,6 +220,7 @@ def _build_pools(self, record, default_pool_name, value_transform_fn):
 
             pools[geo_sets[geo_set]]["values"].append(value)
 
+        defaults.sort()
```

Comment:
Probably best to include a comment on this change as to why it's needed/what it fixes.